### PR TITLE
Add essay about how my career started

### PR DESCRIPTION
**Pull request by salutis**
For a change, this one is a simple story. No sophisticated arguments. 😄

### CODE REVIEW

File: Career.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Back when I was eight or so, my dad used to work as a technician repairing telephones for a national telecom company. One day during his lunch break, he announced that it is time for me to see the place where he works, time to show me around. I had no idea what to expect and little I know that that day would change my life forever.
```
Comment (mikemee) [inline Career.txt]: little _did_ I know

File: Career.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+We left the apartment and I hopped into the back of an old van jam-packed with cables, telephones and various strange looking contraptions. There were no seats in the back so I sat on a pile of tires and held firmly onto various objects that looked like they could provide some stability on my way there.
+
+When we arrived, we passed through, what looked like dozens of, dark corridors inside of the building, finally coming to a big door. When we got in, I could not believe my eyes.
+
+I found myself in a large hall and in front of me was the most interesting object I had ever seen in my life. A telephone switch. It was a huge machine with hundreds of flashing lights, making loud noises filling the hall. There and then, I knew that this is what my life is going to be about. Machines.
```
Comment (mikemee) [inline Career.txt]: Usually it's "Then and there", rather than "There and then", but it's grammatically correct 👍

Comment (mikemee) [reply Career.txt]: what my life _was_ going to be about  (though _is_ is probably correctly, it would usually be _was_)

Comment (salutis) [reply Career.txt]: FYI: I derived "there and then" from "here and now". Tricky!

**Comment by salutis** [comment]
Review feedback addressed, merging. Thank you @mikemee!

**Pull request merged**